x-pack/plugins/aiops/public/components/field_stats_popover/field_stats_popover.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButtonIcon, EuiToolTip } from '@elastic/eui';
+import React, { useCallback, useMemo, useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import { isDefined } from '@kbn/ml-is-defined';
+import {
+  FieldPopover,
+  FieldPopoverHeader,
+  FieldStatsServices,
+  FieldStatsProps,
+} from '@kbn/unified-field-list-plugin/public';
+import type { DataView } from '@kbn/data-views-plugin/common';
+import type { TimeRange as TimeRangeMs } from '@kbn/ml-date-picker';
+import { useEuiTheme } from '../../hooks/use_eui_theme';
+import { FieldStatsContent } from './field_stats_content';
+
+export function FieldStatsPopover({
+  dataView,
+  dslQuery,
+  fieldName,
+  fieldValue,
+  fieldStatsServices,
+  timeRangeMs,
+}: {
+  dataView: DataView;
+  dslQuery?: FieldStatsProps['dslQuery'];
+  fieldName: string;
+  fieldValue: string | number;
+  fieldStatsServices: FieldStatsServices;
+  timeRangeMs?: TimeRangeMs;
+}) {
+  const [infoIsOpen, setInfoOpen] = useState(false);
+  const euiTheme = useEuiTheme();
+
+  const closePopover = useCallback(() => setInfoOpen(false), []);
+
+  const fieldForStats = useMemo(
+    () => (isDefined(fieldName) ? dataView.getFieldByName(fieldName) : undefined),
+    [fieldName, dataView]
+  );
+
+  const trigger = (
+    <EuiToolTip
+      content={i18n.translate('xpack.aiops.fieldContextPopover.descriptionTooltipContent', {
+        defaultMessage: 'Inspect field statistics',
+      })}
+    >
+      <EuiButtonIcon
+        iconType="inspect"
+        onClick={(ev: React.MouseEvent<HTMLButtonElement>) => {
+          setInfoOpen(!infoIsOpen);
+        }}
+        aria-label={i18n.translate('xpack.aiops.fieldContextPopover.topFieldValuesAriaLabel', {
+          defaultMessage: 'Inspect field statistics',
```

Comment:
If you decide to change the tooltip text, change the aria-label too.

Reply:
Updated in ae6e32c2e8b2f3876c6336a1d0a6df424d8e5e94